Shrine/app/src/main/java/com/example/android/authentication/shrine/ui/AuthenticationScreen.kt

```diff
@@ -69,20 +69,33 @@ fun AuthenticationScreen(
     val errorDialogViewModel: ErrorDialogViewModel = hiltViewModel()
 
     // Passing in the lambda / context to the VM
-    val activityContext = LocalContext.current
+    val context = LocalContext.current
     val onSignInWithPasskeysRequest = {
         viewModel.signInWithPasskeysRequest(
             onSuccess = { flag ->
                 navigateToHome(flag)
             },
-        ) { jsonObject ->
-            credentialManagerUtils.getPasskey(
-                context = activityContext,
-                creationResult = jsonObject,
-            )
-        }
+            getPasskey = { jsonObject ->
```

Comment:
is it getPasskeyRequest or reponse or just a result, a bit confusing? Can we name accordingly?